### PR TITLE
Last digit chartws

### PR DESCRIPTION
**Pull request by 4p00rv**
Added digit_chart for last digit prediction trade.

### CODE REVIEW

File: src/javascript/binary/pages/trade/charts/digit_infows.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+        var elem = '<select class="smallfont" name="underlying">';
+        for(i=0;i<underlyings.length;i++){
+            elem = elem + '<option value="'+underlyings[i]+'">'+symbols[underlyings[i]]+'</option>';
+        }
+        elem = elem + '</select>';
+        var contentId = document.getElementById('tab_last_digit-content'),
+            content = '<div class="grd-parent">'+
+                        '<div id="last_digit_histo_form" class="grd-grid-8 grd-grid-mobile-12 grd-centered">'+
+                        '<form class=smallfont action="'+ page.url.url_for('trade/last_digit_info') +'" method="post">'+
+                        text.localize('<div class="grd-grid-mobile-12">Select market :'+ elem +' </div>')+
```
Comment (ashkanx) [inline src/javascript/binary/pages/trade/charts/digit_infows.js]: @4p00rv why are you adding HTML tags to localisation?

Comment (4p00rv) [inline src/javascript/binary/pages/trade/charts/digit_infows.js]: the localisation is for text `Select market :`

Comment (ashkanx) [inline src/javascript/binary/pages/trade/charts/digit_infows.js]: why are you adding <div class="grd-grid-mobile-12">  </div> to it?

**Comment by 4p00rv** [comment]
@ashkanx Fixed the localisation issue, please review.

File: src/javascript/binary/pages/trade/charts/digit_infows.js [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+            if (typeof min_max_counter[val] === 'undefined') {
+                min_max_counter[val] = 0;
+            }
+            min_max_counter[val]++;
+        }
+        var min = Math.min.apply(null, filtered_spots);
+        var max = Math.max.apply(null, filtered_spots);
+        var min_index = filtered_spots.indexOf(min);
+        var max_index = filtered_spots.indexOf(max);
+        console.log(min_index+'-'+max_index);
```
Comment (ashkanx) [inline src/javascript/binary/pages/trade/charts/digit_infows.js]: remove console log

**Pull request merged**